---
layout: page
title: Research
order: 2
---

While an organism’s microbiome plays an essential role in shaping host fitness and phenotype, with significant potential applications in systems such as healthcare or agriculture, our ability to accurately manipulate and predict these interactions and their impacts on the host remains a challenge. Despite continual development and technological advances in the field of microbiome sciences, this knowledge gap has persisted in part because we lack conceptual frameworks to move beyond descriptive work of these systems. My research program broadly aims to fill this gap, specifically by studying the molecular basis of these interactions and, notably, grounding these interactions in existing ecological and evolutionary theory. 
<br>
<br>
My approach to this field has focused on plant-microbe interactions as a tractable model. I use a variety of experimental systems and tools to study these interactions, including amplicon and whole genome sequencing, experimental evolution, <i>in vivo</i> studies, metabolomics, and modeling. Below I highlight some of the broad themes of my work.
<br>
<br>
### Host interactions and evolution with their symbionts: 
With significant microbial impacts on plant fitness and phenotype, one focus of my work is in understanding how interactions with a microbiome shape plant evolution. For example, in my dissertation work, I grew a large genotypic panel of wild-collected <i>Bromus tectorum</i> in the greenhouse that varied in their native salinity habitat. While plants from saline habitats should have been well-adapted  to high salinity, our greenhouse experiment showed that this adaptation could only be observed when plants interacted with microbes ([Ricks et al., 2023](https://www.journals.uchicago.edu/doi/full/10.1086/725393)). Follow up work in a greenhouse experiment with a plant genotypic panel of <i>Brachypodium distachyon</i> collected across a nitrogen gradient, showed that plants were colonized by microbiomes with optimal nitrogen cycling traits for their home environment (Ricks & Raglin., In prep). These experiments challenge classic views on evolution, suggesting that these plant-microbe interactions rather than solely plant traits underlie key adaptations. 

<div class="row">
 <div class="column">
  <img src="/images/saline2.jpg" style="width:55%">
  <img src="/images/canyon1.jpg" style="width:22%">
  <img src="/images/canyon2.jpg" style="width:22%">
  <p style="font-size:10pt;color:DimGrey"> Collection sites for <i>B. tectorum</i> across Utah (Left) Salt flats near Goshen (Middle) Mapleton canyon (Right) Lindon canyon </p>
 </div>
</div>
Some of my current work continues in this area, investigating the specific molecular mechanisms that control these microbiomes and their selective drivers. The plant environment is highly selective for which microbes can colonize ([Ricks & Koide, 2019](https://link.springer.com/article/10.1007/s00442-019-04388-y)), with significant variation between plant species and genotypes. By linking plant microbiomes across a genotypic panel of duckweed with plant genomics and metabolomics, I aim to determine the specific molecular plant interactions that underpin variation in plant microbiome recruitment. Pairing these data with the experimentally derived microbiome effects on plant fitness will allow me to evaluate how selection on these molecular interactions may drive plant evolution. 
<br>
<br>
### Mutualism evolution in host microbiomes
Microbiomes can impact their host fitness through the collection of microbial traits and molecular interactions. These traits, however, are not randomly distributed, but are the result of selection from both the environment and their host. Given interest in beneficial interactions across applied systems, a major throughline of my work aims to frame these microbial interactions in mutualism theory in understanding the selective drivers for beneficial microbiomes.
<br>

For example, some mutualisms are predicted to emerge as an incidental byproduct of a partner’s self-interested activities.  We evaluated whether a byproduct mutualism model could be used to understand the distribution of beneficial plant microbiomes by experimentally evolving soil microbiomes under various watering and plant treatments. We observed that microbiomes that evolved under drought, regardless of history with a plant, mediated plant drought tolerance. These results suggest that beneficial microbes may emerge as an incidental byproduct of microbial adaptation to the environment ([Ricks & Yannarell, 2023](https://royalsocietypublishing.org/doi/pdf/10.1098/rspb.2023.0469)). My current work continues in this area, evaluating the specific molecular mechanisms that may underlie these byproduct traits and their selective drivers across the landscape. Integrating molecular interactions with their evolutionary drivers will further develop how theory can be applied to host-microbiomes.
<br>
<div class="row">
 <div class="column">
  <img src="/images/greenhouse1.jpg" style="width:48%">
  <img src="/images/clover2.jpg" style="width:48%">
  <p style="font-size:10pt;color:DimGrey"> Evaluating the microbial impact on plant drought tolerance </p>
 </div>
</div>
Many mutualistic traits can also be energetically expensive and may therefore only emerge with ensured reciprocal rewards from their partners. When experimentally evolving the model mutualistic bacteria rhizobia under drought, bacteria with traits that mediate drought stress only emerged through host selection (Ricks et al, In prep). While further genomic and experimental analyses have yielded evidence for selection on trehalose production pathways in bacteria that underlie these beneficial traits to the host, framing these interactions in mutualism theory provides the predictive framework for understanding and contextualizing these highly molecular interactions.
<br>
<br>
### Molecular genomics in microbial symbiosis
<figure style="float: right; padding-left: 25px; width: 25%">
 <img src="/images/plates.jpg"/>
 <img src="/images/soil.jpg"/>
 <figcaption style="font-size:9pt;color:DimGrey"> Above: Isolates of the model microbe, <i>Rhizobium leguminosarum</i></figcaption>
 <figcaption style="font-size:9pt;color:DimGrey"> Below: Experimentally evolving populations of <i>Rhizobium leguminosarum</i> in soil</figcaption>
</figure>
While much of our understanding of these microbial systems’ evolution is based on theory built from macroorganisms, the unique genomics of microbes may require a fundamental rethinking here. With largely haploid genomes, mobile genetic elements, and unclear species boundaries, one component of my work aims to better understand how these microorganism’s distinct biology integrates with our evolutionary models. One such area is the involvement of plasmids in microbial coevolution with their host plants. Many microbes form specialized interactions with eukaryotic hosts, from mutualists to pathogens. And while we might expect these traits to be essential components of the microbial genome, they’re often encoded on plasmids, with replication independent of the main bacterial chromosome. Using the legume-rhizobia system, I’m investigating potential evolutionary advantages in encoding symbiosis genes on the mobile elements as well as their impact on mutualism evolution. By sequencing a population of experimentally evolved rhizobia, I’m estimating plasmid movement rates and coevolution with the main replicon.
<br>
<br>
### Scaling microbial evolutionary dynamics to host communities:
<figure style="float: left; padding-right: 25px; width: 25%">
 <img src="/images/duckweed.jpg"/>
 <figcaption style="font-size:9pt;color:DimGrey"> Duckweed community mesocosm, including <i>Spirodela polyrhiza</i>, <i>Lemna minor</i>, & <i>Landoltia punctata</i>  <br> <br> </figcaption>
</figure>
These host-microbial interactions do not exist or evolve in a vacuum, but rather in the complex milieu of host community interactions, landscapes, and ecosystem processes. In some of my current postdoctoral work, I'm investigating how many of the microbial eco-evolutionary processes I describe above can impact plant communities, including their competitive interactions, niche, and range dynamics as well as how these feedback to the evolution of the host-microbe system. I'm addressing these questions by leveraging duckweeds, small aquatic plants, across controlled  plant mesocosms in the laboratory. These mesocosms have been linked with [camera tracking software](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0287739), facilitating high throughput analyses across replicate plant communities in assessing microbial impacts on plant community dynamics.



